### DISCUSSION TITLE
Welcome to laravel-starter Discussions!

### DISCUSSION THREAD

**Discussion by nasirkhan** [nasirkhan/laravel-starter#244] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fduran82** [comment]
hi, great work!, i just run the app on my pc.... some how when you create a new class on scss and compile, nothing happens, never add that class to app.css .... is just my install or you have the same problem?

**Reply by nasirkhan** [reply]
I am not sure if you are editing the right SASS file or not. I would suggest you to look into the [webpack.mix.js](https://github.com/nasirkhan/laravel-starter/blob/master/webpack.mix.js) file and find the right file. 

Just for the reference, I did not developed the themes, rather I integrated those into the project. For the frontend there are two style files and for backend there is one.

**Reply by fduran82** [reply]
the problem is on webpack only the frontend CSS, no big deal, just missing a line and all good. thanks

**Comment by fduran82** [comment]
I found the problem, is only the frontend css in the webpack

**Reply by nasirkhan** [reply]
I am happy that you found the issue and fixed it.

**Reply by Sparfel** [reply]
Hi, can you explain what you do to solve the problem ?
Thanks

**Reply by nasirkhan** [reply]
@Sparfel can you please start a new discussion at https://github.com/nasirkhan/laravel-starter/discussions and explain the issue you are facing?
